src/lib/storage.js

```diff
@@ -9,6 +9,7 @@ import defaultProjectAssets from './default-project';
 class Storage extends ScratchStorage {
     constructor () {
         super();
+        const defaultProjectAssets = defaultProject(this.translator);
         defaultProjectAssets.forEach(asset => this.cache(
```

Comment:
These 2 calls look like they can be deduplicated with one `this.cacheDefaultProject`?